data/p10/attribute_types_obmc.xml

```diff
@@ -697,4 +697,18 @@
         </simpleType>
     </attribute>
 
+    <attribute>
```

Comment:
- SignOff missing in commit message.
- Can you add the following prefix "data: p10: " in the header of the commit message?
- Is it HB only consumed attributes or BMC needs to do some action based on this attribute value?

Reply:
Currently HB will only communicate this attribute to BMC. I think there is work in the future on BMC side to use this attribute.